doc/source/changes/template.rst.inc

```diff
@@ -1,23 +1,21 @@
-﻿.. py:currentmodule:: larray_editor
```

Comment:
are you sure the py:obj references will work without this?

Reply:
Maybe not